pkg/storage/minio/lister.go

```diff
@@ -29,7 +30,12 @@ func (l *storageImpl) List(ctx context.Context, module string) ([]string, error)
 			dict[ver] = struct{}{}
 		}
 	}
+
 	ret := []string{}
+	if len(dict) == 0 {
+		return ret, errors.E(op, fmt.Errorf("Module %s not found", module), errors.KindNotFound)
```

Comment:
Could you use `errors.M(module)` instead of `fmt.Errorf("Module %s not found", module)`?